### DISCUSSION TITLE
Does the pipeline use the first allele or the second allele as reference when reading the .pgen files?

### DISCUSSION THREAD

**Discussion by scienception** [PGScatalog/pgsc_calc#156] [Q&A] [answered]
Is there a flag to specify this or does the software figures it out automatically?

**Comment by smlmbrt** [comment, marked as answer]
The pipeline doesn't use REF/ALT annotations per se, as it can calculate the PGS regardless of whether the effect_allele is the REF or ALT as we allow for both when matching the PGS variants to the genotyping data. In the plink format the 3'rd column of the .pvar file is the REF (https://www.cog-genomics.org/plink/2.0/formats#pvar), similar to VCF.
